### PR TITLE
Token Theming variables

### PR DESCRIPTION
**Pull request by iryanclarke**

### CODE REVIEW

File: data/tokens.json [hunk 1]
```diff
@@ -472,6 +472,39 @@
...
   },
-  "Light": {},
+  "color_tokens": {
+    "text": {
+      "base": {
+        "value": "{colors.grey.800}",
+        "type": "color"
+      },
+      "emphasis": {
+        "value": "{colors.grey.800}",
```
Comment (alexandra-lim) [inline data/tokens.json:482]: Is this and `subdued` meant to have the same value as `base`?

**Review by alexandra-lim — APPROVED** [review]
LGTM!

File: data/tokens.json (hunk 1 above)
Comment (iryanclarke) [reply data/tokens.json]: Updated now :)

**Comment by CCAyl** [comment]
she gucci

**Pull request merged**